### PR TITLE
More on functions

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: content/more-on-functions/reading/functions-as-values/_index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Functions are powerful tools in any programming language, and JavaScript uses these tools in some flexible and creative ways. This chapter introduces a bit more of the power of functions.
+
+## Functions Are Data
+
+We :ref:`defined a value <def-value>` as "a specific piece of data." Some examples are the number `42`, the string `"LC101"`, and the array `["MO", "FL", "DC"]`. *Functions are also values*, and while they appear to be very different from other values we have worked with, they share many core characteristics.
```
Comment (icr8FreeCode) [inline content/more-on-functions/reading/functions-as-values/_index.md:19]: I'm not sure what the intent here was/is are we using coding syntax for normal words? should this be bold or italicized, underlined? 

1. Should this read values (Strings, Arrays, Boolean, null values etc.) are pieces of data that can be assigned to variables. 
2. Also in JavaScript, Functions are Objects so the headers and text Functions as Values should be updated to Functions as Objects or Classes "Functions are also values"

File: content/more-on-functions/reading/functions-as-values/_index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+When we use a variable *name*, we are really using its *value*. If the variable `course` is assigned the value `"LC101"`, then `console.log(course)` prints `"LC101"`. When a variable holds a function, it behaves the same way as when it holds a number or a string. The variable *refers to* the function. 
+
+![The variable helloFunc on the left *referst to* the function hello on the right](pictures/function-var.png?classes=border)
+
+Again, *functions are values*. They can be used just like general values. For example:
+
+- Functions may be assigned to variables.
+- Functions may be used in expressions, such as comparisons.
+- Functions may be converted to other data types.
```
Comment (icr8FreeCode) [inline content/more-on-functions/reading/functions-as-values/_index.md:68]: Please remove the following references as they are incorrect representations of the language 
1. Functions may be converted to other data types.
2. You will probably never need to convert a function to a Boolean, or ask whether a function is greater than 5.

File: content/more-on-functions/reading/functions-as-values/pictures/function-var.png
Comment (icr8FreeCode) [inline content/more-on-functions/reading/functions-as-values/pictures/function-var.png:1]: This image doesn't properly call a JavaScript function and should be removed or updated to hello()

File: content/more-on-functions/reading/pass-function-as-arguments/_index.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Sally Steuterman # to be set by the page reviewer
+reviewerGitHub: gildedgardenia # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+*Functions are data*, and therefore can be passed around just like other values. This means a function can be *passed to another function* as an argument. This allows the *function being called* to use the *function argument* to carry out its action. This turns out to be extremely useful.
```
Comment (icr8FreeCode) [inline content/more-on-functions/reading/pass-function-as-arguments/_index.md:15]: Please replace this with Functions are objects

File: content/more-on-functions/reading/anonymous-functions/pictures/function-var-anonymous.png
Comment (icr8FreeCode) [inline content/more-on-functions/reading/anonymous-functions/pictures/function-var-anonymous.png:1]: This images should also be updated to use the equal sign or call the function add()

**Review by icr8FreeCode — CHANGES_REQUESTED** [review]

File: content/more-on-functions/reading/functions-as-values/_index.md (hunk 1 above)
Comment (jwoolbright23) [reply content/more-on-functions/reading/functions-as-values/_index.md:19]: Updated the textbook to reflect the above changes. Also fixed the missing relref.

File: content/more-on-functions/reading/functions-as-values/_index.md (hunk 2 above)
Comment (jwoolbright23) [reply content/more-on-functions/reading/functions-as-values/_index.md:68]: removed the above references.

File: content/more-on-functions/reading/functions-as-values/pictures/function-var.png
Comment (jwoolbright23) [reply content/more-on-functions/reading/functions-as-values/pictures/function-var.png:1]: removed image from reading

File: content/more-on-functions/reading/pass-function-as-arguments/_index.md (hunk 3 above)
Comment (jwoolbright23) [reply content/more-on-functions/reading/pass-function-as-arguments/_index.md:15]: updated language

File: content/more-on-functions/reading/anonymous-functions/pictures/function-var-anonymous.png
Comment (jwoolbright23) [reply content/more-on-functions/reading/anonymous-functions/pictures/function-var-anonymous.png:1]: removed reference to image and image itself.

**Review by icr8FreeCode — APPROVED** [review]

**Pull request merged**